brian2/devices/cpp_standalone/device.py

```diff
@@ -481,6 +481,21 @@ def variableview_get_with_expression(self, variableview, code,
                                   'standalone scripts.')
 
     def code_object_class(self, codeobj_class=None, pref=None):
+        '''
+        Return `CodeObject` class (either `CPPStandaloneCodeObject` class or input)
+
+        Parameters
+        ----------
+        codeobj_class : None or a CodeObject class.
```

Comment:
The standard way to describe optional parameters in numpy doc is to not mention `None`, but instead write:
```
codeobj_class : a `CodeObject` class, optional
```